.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,21 @@
-<!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
-
 ## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
 
-## Checklist
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+This PR fixes #[fill_in_number_here].
+
+This PR does the following and why...
+
+[Delete this and fill in your response here]
+
+## Essential Checklist
+
 - [ ] The linter/Karma presubmit checks have passed.
-  - These should run automatically, but if not, you can manually trigger them locally using `python -m scripts.pre_commit_linter` and `python -m scripts.run_frontend_tests`.
+- [ ] Allow edits from maintainers is checked, see [this](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for how to enable.
```

Comment:
Put quotes around "Allow edits from maintainers". Also separate the tangential part, viz.:

"Allow edits from maintainers" is checked. (See here for instructions on how to enable it.)

Reply:
done!